src/stores/video.ts

```diff
@@ -145,8 +143,8 @@ export const useVideoStore = defineStore('video', () => {
 
   /**
    * Extracts a thumbnail from the first frame of a video.
-   * @param videoUrl The URL of the video from which to extract the thumbnail.
-   * @returns A promise that resolves with the base64-encoded image data of the thumbnail.
+   * @param {string} videoUrl - The URL of the video from which to extract the thumbnail.
+   * @returns {Promise<string>} A promise that resolves with the base64-encoded image data of the thumbnail.
```

Comment:
This change belongs to the commit before.

A suggestion for this type of situation is to use the `fixup` in git.

This alias is a good one that helps a lot in those cases:

```bash
fixup = "!git log -n 50 --pretty=format:'%h %s' --no-merges | fzf | cut -c -7 | xargs -o git commit --fixup"
```